src/components/ButtonPanel.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropType from 'prop-types';
+import Button from './Button';
+
+const ButtonPanel = ({ id }) => (
+  <div id={id}>
+    <div>
+      <Button>AC</Button>
+      <Button>+/</Button>
+      <Button>%</Button>
+      <Button>÷</Button>
+    </div>
+    <div>
+      <Button>7</Button>
+      <Button>8</Button>
+      <Button>9</Button>
+      <Button>X</Button>
+    </div>
+    <div>
+      <Button>4</Button>
+      <Button>5</Button>
+      <Button>6</Button>
+      <Button>-</Button>
+    </div>
+    <div>
+      <Button>1</Button>
+      <Button>2</Button>
+      <Button>3</Button>
+      <Button>+</Button>
+    </div>
+    <div>
+      <Button>0</Button>
+      <Button>.</Button>
+      <Button>=</Button>
+    </div>
+  </div>
```

Comment:
**[OPTIONAL]** - You could refactor the code here in order to make the rendering of the buttons to be dynamic instead of hardcoded, you could come up with a way to keep your values in a data structure and render the buttons based off on the data structure where you keep your values.

Reply:
Refactored the ButtonPanel component to make it more dynamic, thanks Jorge for the suggestion!